labs/docker-101/4.md

```diff
@@ -1,135 +1,111 @@
 
-  curl localhost:32768
-  172.17.0.1 - - [18/Jan/2019 12:40:17] "GET / HTTP/1.1" 200 -
-  Hello Cisco
-  ```
+     curl localhost:32768
+     172.17.0.1 - - [18/Jan/2019 12:40:17] "GET / HTTP/1.1" 200 -
+     Hello Cisco
+     ```
 
 * ENTRYPOINT [“python”, “-m”, “SimpleHTTPServer”, “8000”]  
-  Like CMD, the ENTRYPOINT command runs a program, but it also does
-  something else: it enables us to pass arguments to the program. We
-  use this feature to pass arguments to Python to start the web server.
+  Like CMD, the ENTRYPOINT command runs a program, but it also does something else: it enables us to pass arguments to the program. We use this feature to pass arguments to     Python to start the web server.
 
-Save the Dockerfile in a new directory and build the new web server
-container image:
+2. Build and run the new web server container image using the following command:
 
-```
-docker build .
-docker run -P <your new container id> &
-```
+    ```
+    docker build .
+    docker run -P <your new container id> &
+    ```
 
-Why don't we see any output?
+    Why don't we see any output?
 
-Every time before when we started a container we saw output from the
-container's application. The reason we see no output this time is that
-our application, a web server, doesn't print any. So how do we know
-whether it's running correctly?
+    Every time before when we started a container we saw output from the container's application. The reason we see no output this time is that our application, a web server,     doesn't print any message. So how do we know whether it's running correctly?
 
-Run the following Docker command:
+3. Run the following Docker command to check if the new web server image is running:
 
-```
-docker ps
-```
-
-The command lists all running containers&mdash;including the new
-web server image that we created:
+    ```
+    docker ps
+    ```
 
-![docker ps](assets/images/dockerpsweb.png)
+    The command lists all running containers including the new web server image that we created:
 
-The web server is running. Now how do we connect to it?
+    ![docker ps](assets/images/dockerpsweb.png)
 
-By default Docker gives each container an internal IP address on the
-host. We can find out our container's IP address by running `docker
-inspect <container ID>`. Running this command displays a great deal of
-information about the container. Included in the output is the
-container's address, identified as "IPAddress":
+    The web server is running. Now how do we connect to it?
 
-![Docker Inspect](assets/images/dockerinspect.png)
+    By default Docker gives each container an internal IP address on the host.
+    
+4. Run the following command to find out the container's IP address.
+    ```
+    docker inspect <container ID>
+    ```
+    Running this command displays a great deal of information about the container. Included in the output is the container's address, identified as "IPAddress".
```

Comment:
Probably need to tell them how to get the container ID, by running `docker ps`.

Reply:
Updated all the review comments. Kindly review...